lab_5_scrapper/scrapper.py

```diff
@@ -127,11 +232,24 @@ def _extract_url(self, article_bs: BeautifulSoup) -> str:
         Returns:
             str: Url from HTML
         """
+        container_div = article_bs.find('div', class_='col-lg-9 col-md-12')
```

Comment:
is there a better strategy to find it?

Reply:
I haven't found a better strategy, on my web site there no classes in a tags and there is no unique class for news container